plugins/development/test/not-reload.test.ts

```
@@ -0,0 +1,37 @@
+import fs from 'node:fs/promises';
+import { scheduler } from 'node:timers/promises';
+
+import { mm, type MockApplication } from '@eggjs/mock';
+import { beforeAll, afterAll, it, describe, afterEach } from 'vitest';
+import { escape, getFilepath, DELAY } from './utils.ts';
```

Biome (2.1.2)

[error] Do not shadow the global "escape" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)